helm-charts/splunk-otel-collector/templates/daemonset.yaml

```
@@ -252,8 +252,14 @@ spec:
         {{- if or $agent.securityContext (and (eq (include "splunk-otel-collector.logsEnabled" $) "true") (eq .Values.logsEngine "otel")) }}
         securityContext:
           {{- if $agent.securityContext }}
-          {{- toYaml $agent.securityContext | nindent 10 }}
-          {{- else }}
+            {{- if .Values.isWindows }}
+              {{- with unset $agent.securityContext "runAsUser" }}
+              {{- toYaml $agent.securityContext | nindent 10 }}
+              {{- end}}
+            {{- else }}
+              {{- toYaml $agent.securityContext | nindent 10 }}  
```

```suggestion
              {{- toYaml $agent.securityContext | nindent 10 }}
```
The trailing whitespace here is the cause of the `pre-commit` [ci failure](https://github.com/signalfx/splunk-otel-collector-chart/actions/runs/5092612755/jobs/9154186535?pr=797).